cmd/background-newdisks-heal-ops.go

```diff
@@ -143,6 +143,26 @@ func initHealingTracker(disk StorageAPI, healID string) *healingTracker {
 	return h
 }
 
+func (h *healingTracker) resetHealing() {
+	h.mu.Lock()
+	defer h.mu.Unlock()
+
+	h.ItemsHealed = 0
+	h.ItemsFailed = 0
+	h.BytesDone = 0
+	h.BytesFailed = 0
+	h.ResumeItemsHealed = 0
+	h.ResumeItemsFailed = 0
+	h.ResumeBytesDone = 0
+	h.ResumeBytesFailed = 0
+	h.ItemsSkipped = 0
+	h.BytesSkipped = 0
```

Comment:
I will be somewhat unintuitive these values reset, but I see your point.

How about separating these stats and have an accumulated as well as one for the current attempt?

Reply:
> How about separating these stats and have an accumulated as well as one for the current attempt?

in fact, you already print those stats (items healed/failed/skipped) for each retry in the standard output.. probably it is not needed to have a separate stats that accumulate everything. By the retry here is a retry from scratch (all buckets and all objects again)

Reply:
Yeah, but it will look weird that suddenly progress goes down to 0 for everything. I don't know if you still do ETA, but that will for sure be useless.

Do we count attempts? That would also be fine by my (and probably easier).

Reply:
oh yes I see what you mean now, yes we send the retry counter information now to mc, I guess we could avoid calculating ETA in mc when we are retrying. By the way, in most use cases, the retry will be much faster because most of the objects will be already healed, the healing will be just listing again and only heal the remaining objects